rules/diff_test.bzl

```diff
@@ -26,17 +26,39 @@ def _runfiles_path(f):
     else:
         return f.path  # source file
 
+def _ignore_line_endings(ctx):
+    ignore_line_endings = "0"
+    if ctx.attr.ignore_line_endings:
+        ignore_line_endings = "1"
+    return ignore_line_endings
+
 def _diff_test_impl(ctx):
     if ctx.attr.is_windows:
+        bash_bin = ctx.toolchains["@bazel_tools//tools/sh:toolchain_type"].path
```

Comment:
Now that we require bash, I wonder if the maintenance of this rule would be simpler, to use the same bash script on windows (and maybe fallback to fc or just use diff)?